test/appSearch.test.js

```diff
@@ -0,0 +1,112 @@
+const {getBearerToken} = require('../src/app/auth')
```

Comment:
Right now we don't have much of tests files, but going forward you'll have dozens of files and it can become a mess. 
I suggest to try thinking of a good structure to organise the files from the start.
Maybe something like this:

test
	apis
		auth
			auth.test.js
		scraper
			appSearch.test.js
			getApp.test.js
	middlewares
		isAutorized.test.js